src/neo/SmartContract/Helper.cs

```diff
@@ -155,40 +155,37 @@ internal static bool VerifyWitnesses(this IVerifiable verifiable, StoreView snap
                     continue;
                 }
 
-                int offset;
-                ContractMethodDescriptor init = null;
-                byte[] verification = verifiable.Witnesses[i].VerificationScript;
-                if (verification.Length == 0)
-                {
-                    ContractState cs = snapshot.Contracts.TryGet(hashes[i]);
-                    if (cs is null) return false;
-                    ContractMethodDescriptor md = cs.Manifest.Abi.GetMethod("verify");
-                    if (md is null) return false;
-                    verification = cs.Script;
-                    offset = md.Offset;
-                    init = cs.Manifest.Abi.GetMethod("_initialize");
-                }
-                else
-                {
-                    if (NativeContract.IsNative(hashes[i])) return false;
-                    if (hashes[i] != verifiable.Witnesses[i].ScriptHash) return false;
-                    offset = 0;
-                }
                 using (ApplicationEngine engine = ApplicationEngine.Create(TriggerType.Verification, verifiable, snapshot?.Clone(), gas))
                 {
                     CallFlags callFlags = verifiable.Witnesses[i].StateDependent ? CallFlags.AllowStates : CallFlags.None;
-                    ExecutionContext context = engine.LoadScript(verification, callFlags, offset);
```

Comment:
The `LoadScript` is removed. But it is not added in `IsNative == true`.

Reply:
![image](https://user-images.githubusercontent.com/3167973/99649863-04707e00-2a55-11eb-8269-6d4c658cc464.png)

It's because it must be empty

Reply:
But you must load the script from ContractState.